### ISSUE TITLE
Change incompatible template button color

### ISSUE THREAD

**Issue by jarekdanielak** [camunda/camunda-modeler#4766]
### What should we do?

The orange button for an incompatible template in properties panel is not meeting the accessibility criteria.

![image](https://github.com/user-attachments/assets/4124470b-12f3-44df-955f-8129b132a510)

### Why should we do it?

Change to the colors in https://github.com/camunda/camunda-modeler/issues/4766#issuecomment-2596279048

### Extra

Suggestions from [this comment](https://github.com/camunda/camunda-modeler/pull/4744#issuecomment-2541828239) could be added along with this.

**Comment by jarekdanielak** [comment]
@lmbateman can you provide more context?

**Comment by lmbateman** [comment]
To be clear, the WCAG contrast success criteria are AA-level, and the Modeler is only required to meet A-level success criteria, so technically, the button meets our contractual requirements. Having said that, fixing it is the right thing to do.

Background: We chose these button colors because:
- The team felt strongly that the button should convey a warning, rather than an error, so we didn't want to use red.
- Other buttons in this family have white text, and we didn't want to change just this button.
- Orange is tricky for accessibility purposes, and the darker oranges tend to read as brown.
- This orange already existed in the Desktop Modeler colorscape, so would harmonize with existing colors. No accessible orange exists.

Going forward, we have a couple of options:
- Find an orange that works.
- Change the text to black or dark gray for just this button (not sure how easy this would be to maintain).

**Comment by barmac** [comment]
> fixing it is the right thing to do.

I agree completely. We want the tools to be accessible for the benefit of our users, and the low bar of target WCAG level is not a reason to keep insufficient contrast.

Regarding the font color change, I think it works fine in GH labels, so should work for us too.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/fa03f08d-d703-4cb0-9f2f-73acfc7086a9" />

**Comment by barmac** [comment]
BTW we may not need to do the CSS magic that GitHub employed though it's tempting ;)
<img width="368" alt="image" src="https://github.com/user-attachments/assets/13c2413b-04b2-4ffb-a87f-f63a46e8e0ca" />

**Comment by BDFL669** [comment]
I'd be happy to help with this task as part of my open source contribution

**Comment by jarekdanielak** [comment]
@BDFL669 thanks for showing interest in contributing! We haven't agreed on exact colors yet, so we can't jump to implementation right away, but please feel free to submit your proposals for the UI here. 😊

**Comment by lmbateman** [comment]
The text color used outside the canvas is #22242a. [It has the following contrast ratios with our oranges](https://docs.google.com/spreadsheets/d/1jR9wTyYUEcm32pH7GJfUxsS86huBwTNCLra9GCd6gGI/edit?usp=sharing):

<img width="853" alt="Image" src="https://github.com/user-attachments/assets/abb04f4c-3c10-426f-af85-29d01fd08c0a" />

So we can use the current orange and the standard text color to fix the accessibility issue. (I believe the variable name for the standard text color is --color-grey-225-10-15.)

**Comment by barmac** [comment]
Thanks for your assessment. I've updated the top-comment.

**Comment by jarekdanielak** [comment]
Fixed upstream in [bpmn-io/bpmn-js-element-templates@2.5.0](https://github.com/bpmn-io/bpmn-js-element-templates/releases/tag/v2.5.0)

**Comment by philippfromme** [comment]
Closed via https://github.com/camunda/camunda-modeler/pull/4810.
